sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs

```diff
@@ -549,12 +549,12 @@ public partial class DnsMXRecord
     public partial class DnsNSRecord
     {
         public DnsNSRecord() { }
-        public string NsdName { get { throw null; } set { } }
+        public string DnsNsdName { get { throw null; } set { } }
     }
     public partial class DnsPtrRecord
     {
         public DnsPtrRecord() { }
-        public string PtrdName { get { throw null; } set { } }
+        public string DnsPtrdName { get { throw null; } set { } }
```

Comment:
Should be `Ptr`, what is `Ptrd`?

Reply:
The name of the [swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/dns/resource-manager/Microsoft.Network/stable/2018-05-01/dns.json#L1209) definition is prtdname

Reply:
Ok, according to the swagger, the name should be `PtrDomainName`